lib/client/email/fb-email-client.unit.spec.js

```diff
@@ -0,0 +1,142 @@
+const test = require('tape')
+const {stub, useFakeTimers} = require('sinon')
+
+const jwt = require('jsonwebtoken')
+
+const FBEmailClient = require('./fb-email-client')
+
+/* test values */
+const userId = 'testUserId'
+const serviceSlug = 'testServiceSlug'
+const serviceToken = 'testServiceToken'
+const serviceSecret = 'testServiceSecret'
+const emailUrl = 'https://email'
+const createEndpointUrl = `${emailUrl}/service/${serviceSlug}/user/${userId}`
+const setEndpointUrl = createEndpointUrl
+const payload = {foo: 'bar'}
+
+// Ensure that client is properly instantiated
+
+/**
+ * Convenience function for testing client instantiation
+ *
+ * @param {object} t
```

Comment:
could this be names `type` instead of just `t`?

Reply:
The convention is `test('When....', t => { ... } )` and it's passing that `t` instance rather than the `tape` itself